wp-content/themes/oceanwp-child/single-blog_post.php

```diff
@@ -17,32 +17,32 @@
 
 	<style>
 		.single .entry-title{
-    display:none;
-}
-.wpulike-heart .wp_ulike_general_class{
-    background: linear-gradient(80.63deg, #0B4F6D 26.04%, #0F7AA9 99.31%) !important;
-    box-shadow: 2px 3px 9px rgba(0, 0, 0, 0.2) !important;
-    border-radius: 30px !important;
-}
-
-.wpulike-heart .count-box{
-    color: #fff !important
-}
-
-.ess-all-networks-button, #ess-main-wrapper .ess-total-share{
-    background: linear-gradient(80.63deg, #0B4F6D 26.04%, #0F7AA9 99.31%) !important;
-    box-shadow: 2px 3px 9px rgba(0, 0, 0, 0.2) !important;
-    border-radius: 30px !important;
-}
-
-@media only screen and (min-width: 960px){
-    .content-area, .content-left-sidebar .content-area{
-        width:75%
-    }
-    .widget-area.sidebar-primary {
-        width: 25%;
-    }
-}
+			display:none;
+		}
+		.wpulike-heart .wp_ulike_general_class{
+			background: linear-gradient(80.63deg, #0B4F6D 26.04%, #0F7AA9 99.31%) !important;
+			box-shadow: 2px 3px 9px rgba(0, 0, 0, 0.2) !important;
+			border-radius: 30px !important;
+		}
+
+		.wpulike-heart .count-box{
+			color: #fff !important
+		}
+
+		.ess-all-networks-button, #ess-main-wrapper .ess-total-share{
+			background: linear-gradient(80.63deg, #0B4F6D 26.04%, #0F7AA9 99.31%) !important;
+			box-shadow: 2px 3px 9px rgba(0, 0, 0, 0.2) !important;
+			border-radius: 30px !important;
+		}
```

Comment:
I think I've created a `child-plugin` for this plugin. Maybe we should add this style there?

Reply:
In the layout page itself or you want me to create separate style sheet for it?

Reply:
Let's start with the layout page. If qe see we need more tweaks and variants we can create a file.

Reply:
I have made changes to this one but it seems to be not using the child plugin as the changes didn't take effect, so I had to make the changes in the ess plugin itself. That worked.

Reply:
Okay reverting changes for this as I'm using the share button in initiatives as well with different design and adding this style in the layout file over rides my initiative share button styling. Let's leave it in the single blog page.